formats/json-jackson/src/test/java/io/cloudevents/jackson/JsonFormatTest.java

```diff
@@ -120,6 +123,29 @@ void throwExpectedOnInvalidSpecversion() {
             .hasMessageContaining(CloudEventRWException.newInvalidSpecVersion("9000.1").getMessage());
     }
 
+    @ParameterizedTest
+    @MethodSource("badJsonContent")
+    /**
+     * JSON content that should fail deserialization
+     * as it represents content that is not CE
+     * specification compliant.
+     */
+    void verifyDeserializeError(String inputFile){
+
+        byte[] input = loadFile(inputFile);
+        boolean exceptionThrown = false;
+
+        try {
+
+            CloudEvent ce = getFormat().deserialize(input);
+
+            Assertions.fail("Expected Exception did not occur");
+
+        } catch (EventDeserializationException ede){
+        }
+
```

Comment:
Can you use assertj assertions for failures? https://assertj.github.io/doc/#assertj-core-exception-assertions-assertThatThrownBy

Reply:
Ack